pkg/monitortests/testframework/disruptionexternalcloudservicemonitoring/cloudmonitortest.go

```diff
@@ -0,0 +1,90 @@
+package disruptionexternalcloudservicemonitoring
+
+import (
+	"context"
+	_ "embed"
+	"time"
+
+	"github.com/openshift/origin/pkg/monitortestframework"
+	"github.com/openshift/origin/pkg/monitortestlibrary/disruptionlibrary"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/origin/pkg/monitor/backenddisruption"
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+)
+
+const (
+	newCloudConnectionTestName    = "[sig-trt] disruption/ci-cloud-network-liveness connection/new should be available throughout the test"
```

Comment:
gcp-connectivity
gcp-external-connectivity
gcp-network-liveness
gcp-static-connectivity

What do we want here? Something gcp specific.